server/controllers/feedBack/updateFeedBack.js

```diff
@@ -0,0 +1,27 @@
+const FeedBack = require('../../models/feedBack');
+const { addFeedBackSchema } = require('../../utils/validation');
+
+const updataFeedBack = async (req, res, next) => {
+  try {
+    const { commentId } = req.params;
+    const { comment, star } = req.body;
+
+    const validUpdate = await addFeedBackSchema.validateAsync({
+      comment, star,
+    }, { abortEarly: false });
+
+    const updatedData = FeedBack.update({ commentId, ...validUpdate }, {
```

Comment:
- you must be sure the data is exist before update it (you can use findByPk)
- update query is returns an array with one or two elements, use ( returning: true) option to be sure the array have two items
- make the error custome with its state (you can see updateMessage.js file on 15-biuld-database branch)